### DISCUSSION TITLE
dynaframe4 meets magicMirror2

### DISCUSSION THREAD

**Discussion by fakorte** [Geektoolkit/Dynaframe3#99] [Ideas]
Dynaframe works like a dream for displaying photos and videos. The design and the variety of information of [MagicMirror ](https://magicmirror.builders/)is also awesome. Couldn't some of the source code be used in this project? The result would be a great picture frame with a lot of edge information that can be displayed beautifully.

**Comment by Geektoolkit** [comment]
I am thinking on the scenario...it'd be similar to dakboard. I started out using magic mirror, and while I liked it for what it was, it was very 'techie' for what I wanted to achieve...I found myself contstantly in config files to add stuff, and had alot of issues with addons that were no longer supported.  That being said, it's got an amazing addon community behind it that have done a ton of work for making importers.

To have that data show up on top of the images in dynaframe would require careful thought to make sure that the text didn't get washed out...so like a backing grey box or something. I'm thinking about the scneairo but it'd likely require a developer passionate about making that happen to jump in to architect that in a reliable way.  

That being said, I do plan on doing a 'webview' for this that would make it not only scroll images and videos, but webpages. Those webpages could be served by magic mirror and so you'd have a way of showing it occasionally lets say.
